diff -aurN libjxl-0.5/deps.sh libjxl-0.5-fixed/deps.sh
--- libjxl-0.5/deps.sh	2021-08-02 09:20:01.000000000 -0700
+++ libjxl-0.5-fixed/deps.sh	2021-09-19 21:18:08.743223100 -0700
@@ -13,6 +13,7 @@
 
 # Git revisions we use for the given submodules. Update these whenever you
 # update a git submodule.
+THIRD_PARTY_BROTLI="35ef5c554d888bef217d449346067de05e269b30"
 THIRD_PARTY_HIGHWAY="e2397743fe092df68b760d358253773699a16c93"
 THIRD_PARTY_LODEPNG="48e5364ef48ec2408f44c727657ac1b6703185f8"
 THIRD_PARTY_SKCMS="64374756e03700d649f897dbd98c95e78c30c7da"
@@ -69,6 +70,7 @@
   fi
 
   # Sources downloaded from a tarball.
+  download_github third_party/brotli google/brotli
   download_github third_party/highway google/highway
   download_github third_party/lodepng lvandeve/lodepng
   download_github third_party/sjpeg webmproject/sjpeg
